src/views/about/about.jsx

```diff
@@ -60,6 +60,14 @@ var About = React.createClass({
                             <h3><FormattedMessage id='about.aroundTheWorld' /></h3>
                             <img src="/images/about/around-the-world.png" alt="" />
                             <p><FormattedHTMLMessage id='about.aroundTheWorldDescription' /></p>
+                            values={{
```

Comment:
This should be FormattedMessage (not FormattedHTMLMessage) 
The placeholder in `'about.aroundTheWorldDescription'` is `{translationLink}`, so the values object just needs to specify the value for `translationLink`
```jsx
values={{
    translationLink: (
        <a href='http://wiki.scratch.mit.edu/wiki/How_to_Translate_Scratch\'>
            <FormattedMessage id='about.translationLinkText' />
        </a>
    )
}}
```
That is, the name in the placeholder becomes the key in the values object, and the value is the link along with the translation of the link text.

Reply:
Okay 👍